_editorial_policies/02_reviewing.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: single
+sidebar:
+  nav: reviewing.md
+title: Policies About Review of Manuscripts
+excerpt: A draft of the policies for reviewing manuscripts
+permalink: /policies/reviewing/
+---
+
+## Conflicts of interest
+
+A reviewer may not have conflict of interest with an author of paper
```

Comment:
See overall comments on "conflict of interest with".

Reply:
And this looks like the second COI section

Reply:
Will figure out how to link back to sections.